src/dodal/beamlines/p99.py

```diff
@@ -32,3 +44,30 @@ def sample_stage() -> XYZPositioner:
 @device_factory()
 def lab_stage() -> XYZPositioner:
     return XYZPositioner(f"{PREFIX.beamline_prefix}-MO-STAGE-02:LAB:")
+
+
+set_path_provider(
+    StaticVisitPathProvider(
+        BL,
+        Path("/dls/p99/data/2024/cm37284-2/processing/writenData"),
+        client=LocalDirectoryServiceClient(),  # RemoteDirectoryServiceClient("http://p99-control:8088/api"),
+    )
+)
+
+
+@device_factory()
+def andor2_det() -> Andor2Detector:
+    """Andor model:DU897_BV."""
+    return Andor2Detector(
+        prefix=f"{PREFIX.beamline_prefix}-EA-DET-03:",
+        path_provider=get_path_provider(),
+        drv_suffix="CAM:",
+        fileio_suffix="HDF5:",
```

Comment:
nit: both of these are available from `dodal.common.beamlines.device_helpers` as `CAM_SUFFIX` and `HDF5_SUFFIX` respectively